From d21d27260b98325fc5592116f50355a9a5391e3f Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 22 Apr 2022 17:37:23 +0200
Subject: [PATCH 138/N] CI: remove --without-c-locale-coercion

No longer needed since the default now works on Windows
---
 .github/workflows/mingw.yml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/.github/workflows/mingw.yml b/.github/workflows/mingw.yml
index 999ccf9..40b1d5e 100644
--- a/.github/workflows/mingw.yml
+++ b/.github/workflows/mingw.yml
@@ -75,7 +75,6 @@ jobs:
             --with-system-ffi \
             --with-system-libmpdec \
             --without-ensurepip \
-            --without-c-locale-coercion \
             --enable-loadable-sqlite-extensions \
             --with-tzpath=${MINGW_PREFIX}/share/zoneinfo \
             --enable-optimizations
@@ -187,7 +186,6 @@ jobs:
             --with-system-ffi \
             --with-system-libmpdec \
             --without-ensurepip \
-            --without-c-locale-coercion \
             --enable-loadable-sqlite-extensions
 
           make -j8
-- 
2.36.1

